mbed_host_tests/host_tests/base_host_test.py

```diff
@@ -49,13 +49,13 @@ def notify_complete(self, result=None):
         if self.__event_queue:
             self.__event_queue.put(('__notify_complete', result, time()))
 
-    def reset_dut(self):
+    def reset_dut(self, value):
```

Comment:
Yes. It is a breaking change. Please see the corresponding PR for greentea-client test https://github.com/ARMmbed/greentea-client/pull/10.